app/assess/routes.py

```diff
@@ -543,8 +544,18 @@ def sub_crit_scoring():
 @login_required(roles_required=["LEAD_ASSESSOR"])
 def resolve_flag(application_id):
     form = ResolveFlagForm()
-    # TODO: Resolve flag for multiple sections flag to be implemented
-    section = [request.args.get("section_id", "section not specified")]
+    flag_id = request.args.get("flag_id")
+
+    if not flag_id:
+        abort(404)
```

Comment:
worth logging here?
abort(400, "No flag id found")

Reply:
It takes me to unformatted page if I use `abort(400, "No flag id found")`. 
![image](https://github.com/communitiesuk/funding-service-design-assessment/assets/127315890/4d29c9ba-ea64-4e02-85d9-97d853741c0f)